app/jobs/replicated_file_check_job.rb

```diff
@@ -10,13 +10,24 @@ class ReplicatedFileCheckJob < ApplicationJob
 
   # @param [PreservedCopy] verify that the archived preserved_copy exists on an endpoint
   def perform(preserved_copy)
-    return if preserved_copy.unreplicated? # FIXME: storytime (Monday 07/02)
+    if preserved_copy.unreplicated?
+      Rails.logger.error("Archival Preserved Copy: #{preserved_copy} has a status of #{preserved_copy.status}.")
```

Comment:
Can we log a hint about why this is an error? Maybe, "`#{preserved_copy} should be replicated, but has status of #{preserved_copy.status}`", or something similar?

Reply:
Don't need to interpolate since by the conditional we already know the status is `unreplicated`.